hsfs/java/beam/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It contains java code also, is it intended?

Reply:
Yes, Davit said to move java code too.